src/containters/Home/components/HeroView/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import './index.scss';
+import { IoIosArrowForward } from 'react-icons/io';
+import { BrowserRouter as Router, Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+const HeroView = (props) => {
+    const { heroViewData } = props
+    return (
+        <Router>
```

Comment:
- Please remove this `Router` component as it has no use the `Link` component will work normally without it.